webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js

```diff
@@ -1,43 +1,59 @@
 import React from 'react';
+import { Button } from '@patternfly/react-core';
 import EmptyStatePattern from './EmptyStatePattern';
-import PrimaryActionButton from './EmptyStatePrimaryActionButton';
-import SecondaryActionButtons from './EmptyStateSecondaryActionButtons';
 import { defaultEmptyStatePropTypes } from './EmptyStatePropTypes';
-import { translate as __ } from '../../../common/I18n';
+import { noop } from '../../../common/helpers';
 
 const DefaultEmptyState = props => {
   const {
     icon,
     iconType,
     header,
     description,
-    documentation: {
-      url,
-      label = __('For more information please see'),
-      buttonLabel = __('Documentation'),
-    } = {},
+    documentation,
     action,
     secondaryActions,
   } = props;
 
-  const documentationBlock = url ? (
-    <React.Fragment>
-      {label}{' '}
-      <a href={url} target="_blank" rel="noopener noreferrer">
-        {buttonLabel}
-      </a>
-    </React.Fragment>
+  const actionButtonClickHandler = ({ url, onClick = noop }) => {
+    onClick();
+    if (url) {
+      window.location.href = url;
```

Comment:
@MariaAga did you mention before that using `location.href` had issues in tests?

Reply:
From what I remember there were 2 uses for `window.location` in the same component so mocking for both cases wasn't possible. Maybe here it can be mocked easily

Reply:
@rabajaj0509 can you add a small test for that link?
need to mock that `location.href` method, trigger a `click` on the link,
and expect the mock method to be called with the link address